modules/docs/mdx/CONTRIBUTING.mdx

```diff
@@ -131,18 +113,49 @@ fix: Add missing static class variable to IconButton and Avatar
 
 #### Pull Requests
 
-- All development should happen on a personal fork (including maintainers). This allows us to keep
-  the project tree clean and easy to follow, while ensuring everyone has the same developer
-  experience.
-- Project maintainers are able to push commits to fork PRs, but other contributors will need to open
```

Comment:
Moved this down, it's not important and didn't make sense when I read it, so it's also updated below.